description/src/description.rs

```diff
@@ -8,12 +8,19 @@ use crate::transformer::Transformer;
 
 use super::formatting::format_type_description;
 
-pub fn type_description(type_id: u32, type_registry: &PortableRegistry) -> anyhow::Result<String> {
+/// if format is enabled, `format_type_description` is applied to the end result.
```

Comment:
If this is exposed, I would suggest having more explicit docs for it. Something along the lines of 'Describes the given type ID..' or similar